dashboard/src/actions/operators.test.tsx

```diff
@@ -193,14 +194,15 @@ describe("getCSV", () => {
 
 describe("createResource", () => {
   it("creates a resource", async () => {
-    Operators.createResource = jest.fn(() => true);
+    const resource = {} as IResource;
+    Operators.createResource = jest.fn(() => resource);
     const expectedActions = [
       {
         type: getType(operatorActions.creatingResource),
       },
       {
         type: getType(operatorActions.resourceCreated),
-        payload: true,
+        payload: resource,
```

Comment:
The fact that this can change means the test isn't actually verifying the resource, I think? But fine either way.

Reply:
yep, since the method is mocked the no validation is made